erpnext/templates/deployment-erpnext.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       serviceAccountName: {{ include "erpnext.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
-      {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled }}
+      {{- if and .Values.volumePermissions.enabled .Values.persistence.worker.enabled }}
```

Comment:
`.Values.volumePermissions.enabled` seems to be a string and not a boolean? (no/yes) If that is the case first part of the the condition will always be true?